ServerCore/Pages/Events/CreateDemo.cshtml

```diff
@@ -28,25 +28,33 @@
                 <input asp-for="Event.AnswerSubmissionEnd" class="form-control" />
                 <span asp-validation-for="Event.AnswerSubmissionEnd" class="text-danger"></span>
             </div>
+            <div class="form-group">
+                <div class="checkbox">
+                    <label>
+                        <input asp-for="AddCreatorToLoneWolfTeam" /> @Html.DisplayNameFor(model => model.AddCreatorToLoneWolfTeam)
```

Comment:
Do you know what this defaults to? It would be nice if we could default to yes since I'm guessing the common test case is to check on puzzles from a player's perspective rather than to test team registration from a player's perspective

Reply:
So far it's defaulting to false just like the "start the event" checkbox. Can turn these to true later if the group prefers.